status/raw/status-app-protocols.md

```diff
@@ -0,0 +1,372 @@
+For the sake of illustration, consider the following common content topic and filter use cases:
+- all messages belonging to the same 1:1 chat are always filtered together and therefore SHOULD use the same content topic (see [55/STATUS-1TO1-CHAT](../55/1to1-chat.md))
+- all messages belonging to the same Community are always filtered together and therefore SHOULD use the same content topic (see [56/STATUS-COMMUNITIES](../56/communities.md)).
+
+The app-level content topic MUST be populated in the `content_topic` field in the encapsulating Waku message (see [Waku messages](#waku-messages)).
+
+### Ephemerality
+
+Each Status app-level message MUST define its _ephemerality_.
+Ephemerality is a boolean value, set to `true` if a message is considered ephemeral.
+Ephemeral messages are messages emitted by the app that are transient in nature.
+They only have temporary "real-time" value
+and SHOULD NOT be stored and retrievable from historical message stores and sync caches.
+Similarly, ephemeral message delivery is best-effort in nature and SHOULD NOT be considered in message reliability mechanisms (see [End-to-end reliability layer](#end-to-end-reliability-layer)).
+
+An example of ephemeral messages would be periodic status update messages, indicating a particular user's online status.
+Since only a user's current online status is of value, there is no need to store historical status update messages.
+Since status updates are periodic, there is no strong need for end-to-end reliability as subsequent updates are always to follow.
+
+App-level messages that are considered ephemeral, MUST set the `ephemeral` field in the encapsulating Waku message to `true` (see [Waku messages](#waku-messages))
+
+## End-to-end reliability layer
+
+The end-to-end reliability layer contains the functions related to one of the two end-to-end reliability schemes defined for Status app messages:
+1. Minimum Viable protocol for Data Synchronisation, or MVDS (see [STATUS-MVDS-USAGE](./status-mvds.md))
+2. Scalable distributed log reliability (spec and a punchier name TBD, see the [original forum post announcement](https://forum.vac.dev/t/end-to-end-reliability-for-scalable-distributed-logs/293/16))
+
+Ephemeral messages SHOULD omit this layer.
+Non-ephemeral 1:1 chat messages SHOULD make use of MVDS to achieve reliable data synchronisation between the two parties involved in the communication.
+Non-ephemeral private group chat messages build on a set of 1:1 chat links
+and consequently SHOULD also make use of MVDS to achieve reliable data synchronisation between all parties involved in the communication.
+Since MVDS does not scale for large number of participants in the communication,
+non-ephemeral community messages MUST use scalable distributed log reliability as defined in this [original forum post announcement](https://forum.vac.dev/t/end-to-end-reliability-for-scalable-distributed-logs/293/16).
+The app MUST use a single channel ID per community.
```

Comment:
Do we want to mention that non-ephemeral 1:1 chat messages MAY use e2e reliabilty protocol in the future?

Reply:
Yes! Done. [9ce3bc5](https://github.com/vacp2p/rfc-index/pull/105/commits/9ce3bc5700324e9172729808b4af9eb68d9a9a91)